### ISSUE TITLE
consider creating a separate package from cli

### ISSUE THREAD

**Issue by paulcpederson** [node-swig/swig#14]
This is a pattern I've seen more and more people start to use in the npm community. Essentially there would be `swig-cli` which would do all of the command line compiling/minification. 

This would enable `swig` to have zero dependencies.

**Comment by nylen** [comment]
Good idea, and I can probably help with this.  @paulcpederson do you know if anyone is still active in this repo?

**Comment by paulcpederson** [comment]
Nobody has been actively working on it, but I'd love to pick it back up again. @cdaringe probably feels the same.

**Comment by cdaringe** [comment]
yep.  i'm in SE @ WAC for a couple hours hackin if you want to meet up
